frontend/src/content/messages.js

```diff
@@ -3,6 +3,7 @@ export default {
   tooManyInteractionPartner: 'There are too many interaction partners to be displayed, use our API to get the results.',
   modelNotFound: 'The url contains a model identifier that does not exist in the database. Browse the available models on the Explore page.',
   mapNotFound: 'Sorry, it seems that the selected 2D map is not available because the SVG is missing.<br>If you think this is an accident, drop us a line at <a href="mailto:contact@metabolicatlas.org?subject=SVG error&body=There was an error when choosing to see the 2D map for ..">contact@metabolicatlas.org</a>.',
+  maIDNotFound: 'It looks like this identifier is no longer part of any of the models integrated into our website. For more details about this id please look <a href="https://github.com/SysBioChalmers/Human-GEM/tree/main/data/deprecatedIdentifiers">here</a>.',
```

Comment:
```suggestion
  maIDNotFound: 'It looks like this identifier is no longer part of any of the models integrated into our website. For more details about deprecated identifiers please look <a href="https://github.com/SysBioChalmers/Human-GEM/tree/main/data/deprecatedIdentifiers">here</a>.',
```

Reply:
It makes more sense. I've changed it in f0414776